shell/display.cc

```diff
@@ -170,7 +170,7 @@ void Display::registry_handle_global(
 
   else if (strcmp(interface, wl_output_interface.name) == 0) {
     d->m_output = static_cast<struct wl_output*>(
-        wl_registry_bind(registry, name, &wl_output_interface, 1));
+        wl_registry_bind(registry, name, &wl_output_interface, 2));
```

Comment:
For the versioning we usually do MIN(2, version).

Reply:
noted and addressed